blueprints/node-webkit/index.js

```diff
@@ -23,8 +23,9 @@ module.exports = {
     return this.addNwConfig(options)
       .then(function() {
         if (!dependencies.nw) {
-          return _this.addPackageToProject('nw');
+          _this.addPackageToProject('nw');
```

Comment:
[`addPackageToProject`](https://github.com/ember-cli/ember-cli/blob/2e64faa354f9d4f2a1c0202f91bd5f325345af2e/lib/models/blueprint.js#L845) returns a promise. I would keep the `return` statement here.

In general, try to chain promises, not nest them.